classifier.py

```diff
@@ -295,7 +295,9 @@ class GradeLevelClassifier(Classifier):
 
     @classmethod
     def audience(cls, identifier, name, require_explicit_age_marker=False):
-        young, old = cls.target_age(identifier, name, require_explicit_age_marker)
+        target_age = cls.target_age(identifier, name, require_explicit_age_marker)
```

Comment:
This was a bug created when I switched from `target_age` always returning a 2-tuple to making it always return a NumericRange.